libmamba/include/mamba/core/satisfiability_error.hpp

```diff
@@ -70,31 +77,14 @@ namespace mamba
         };
         struct PackageNode : PackageInfo
```

Comment:
Why keeping empty types? If you find the name more expressive, you can alias the type (although it won't prevent to pass the real type to functions accepting the alias).

Reply:
Strong typing. They are put in a `std::variant` so they cannot be the same type.

Reply:
Oh, indeed.